index.html

```diff
@@ -85,13 +85,23 @@ <h3>Party</h3>
             <a href="" class="all-program">SEE THE WHOLE PROGRAM</a>
         </section>
 
-        <section class="speakers-container">
-            <h2>Featured Speakers</h2>
+        <div id="nav" class="nav-transparent">
+            <h2 class="speaker-header">Featured Speakers</h2>
             <span class="hr"></span>
-            <div id="speaker-holder">
-            </div>
-        </section>
+            <!-- Your navigation content here -->
+        </div>
+
+        <div id="speaker-holder" class="speaker-container">
+            <!-- Speaker items will be dynamically added here -->
+        </div>
```

Comment:
- [ ] As highlighted by the previous reviewer, it's important to ensure that all the speaker or actor cards are centered on the screen. To achieve this, consider adding appropriate padding to the container of the cards. This adjustment will help maintain a visually pleasing and balanced layout for your project. Keep up the good work! 🌟📝🚀
![image](https://github.com/amiralda/haitian-Otaku-Website/assets/55840999/de8e0a71-e87c-455f-9c3f-b316ca9f4ecb)